BUILD.bazel

```diff
@@ -247,3 +247,34 @@ proto_library(
         "@com_google_protobuf//:wrappers_proto",
     ],
 )
+
+proto_library(
```

Comment:
Not sure if Java libraries are needed but this at least gets Bazel building the s2a protos.

Reply:
We should be able to remove the java_proto_library from the other ones. No need to add it here.